docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -53,26 +53,24 @@ Now, let's tell the `native-image` executable about this configuration file by t
 
 [source,xml]
 ----
-<plugin>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-maven-plugin</artifactId>
-    <version>${version.quarkus}</version>
-    <executions>
-        <execution>
-            <goals>
-                <goal>native-image</goal>
-            </goals>
-            <configuration>
-                <enableHttpUrlHandler>true</enableHttpUrlHandler>
-                <additionalBuildArgs>
-                    <additionalBuildArg>-H:ResourceConfigurationFiles=${project.basedir}/resources-config.json</additionalBuildArg>
-                </additionalBuildArgs>
-            </configuration>
-        </execution>
-    </executions>
-</plugin>
+<profiles>
+    <profile>
+        <id>native</id>
+        <properties>
+            <quarkus.package.type>native</quarkus.package.type>
+            <quarkus.native.additional-build-args>-H:ResourceConfigurationFiles=${project.basedir}/resources-config.json</quarkus.native.additional-build-args>
+        </properties>
+    </profile>
+</profiles>
 ----
 
+[TIP]
+====
+Multiple options should be separated by a comma.
```

Comment:
```suggestion
Multiple options can be separated by a comma.
```

Reply:
IMHO "should" is correct here, since you must use commas in this case

Reply:
I agree with @gastaldi

Reply:
Yes. When I made the suggestion I found `should` a bit high toned for a tip hence the suggestion.  But thinking about it, the original sentence was right as @gastaldi said. Let's keep it that way.

Reply:
Actually `must` is the right term here, for the same reason